### ISSUE TITLE
Using "body" as (custom) CSS Class name produces invalid Premailer.net output

### ISSUE THREAD

**Issue by driekus77** [milkshakesoftware/PreMailer.Net#105]
Hi All,

It took me a while to figure this one out that's why I'm posting it as an issue. Its sort of lame to use "body" as CSS class name but one of my tests did. And the input html works fine in all browsers and even the Ruby Premail seems to handle it oke. But the Premailer.NET seems to use the wrong classes to put background color and width on the table.body element.
After renaming in the input html the body class to bbody everything works oke.

Attached my input html for both wrong and working versions.
[drip test.zip](https://github.com/milkshakesoftware/PreMailer.Net/files/222084/drip.test.zip)

Kind regards,

Henry Roeland

**Comment by martinnormark** [comment]
Can you post an example of the premailed output that is messed up?

**Comment by driekus77** [comment]
Hi @martinnormark,

I have attached a zip file to this post which contains both the drip.input.html as input file and the drip.output.html as messed up output file.

What I needed to do in order to get it working oke was:
- 	Rename CSS Class "body" to "bodyClass"
- 	Line 794: margin/Margin: Changed from 0 to auto to center the table.

Thanks for the help.

Kind regards,

Henry Roeland

[drip test.zip](https://github.com/milkshakesoftware/PreMailer.Net/files/222588/drip.test.zip)
